### DISCUSSION TITLE
Disable pagination all

### DISCUSSION THREAD

**Discussion by ricksieper** [filamentphp/filament#11291] [Help] [answered]
### Package

Table builder

### Package Version

v3.2.24

### How can we help you?

Hello

I just migrated my application from the V2 to the V3.2

I used this function **getTableRecordsPerPageSelectOptions** to custom all the pagination

But I saw it is now empty and no alternative
I saw how to do it in one place [https://filamentphp.com/docs/3.x/tables/advanced#pagination](url) but I have more than 10 lists it will be hard to manage it and each time I create one Ican forget to custom it

There is a a solution or it's now impossible 

thanks for your help

**Comment by leandrocfe** [comment, marked as answer]
```php
//app/Providers/AppServiceProvider.php

use Filament\Tables\Table;

public function boot(): void
{
    Table::configureUsing(function (Table $table) {
        $table->paginated([10, 25, 50, 100]);
    });
}
```

**Reply by ricksieper** [reply]
Perfect thank you
